doc/source/drivers/vector/idb.rst

```diff
@@ -41,11 +41,25 @@ Driver capabilities
 Environment variables
 ---------------------
 
--  **INFORMIXDIR**: It should be set to Informix client SDK install dir
--  **INFORMIXSERVER**: Default Informix server name
--  **DB_LOCALE**: Locale of Informix database
--  **CLIENT_LOCALE**: Client locale
--  **IDB_OGR_FID**: Set name of primary key instead of 'ogc_fid'.
+-  .. config:: INFORMIXDIR
+
+      It should be set to Informix client SDK install dir
+
+-  .. config:: INFORMIXSERVER
+
+      Default Informix server name
+
+-  .. config:: DB_LOCALE
+
+      Locale of Informix database
+
+-  .. config:: CLIENT_LOCALE
+
+      Client locale
+
+-  .. config:: IDB_OGR_FID
```

Comment:
IDB_OGR_FID is actually the only GDAL configuration option. The 4 above do not appear in GDAL sources, and must thus be real environment variables of the informix lib

Reply:
This prompted me to grep out all of the options from the docs and check them against the source; there are about about a dozen that I don't immediately see. I'll look into the following:

GDAL_VRT_TRUSTED_MODULES
GDAL_PDF_DPI
GDAL_PDF_NEATLINE
GDAL_USER_PWD
GDAL_PDF_BANDS
GDAL_PDF_LAYERS
GDAL_PDF_LAYERS_OFF
CARTO_PAGE_SIZE
LIBKML_USE_SCHEMADATA
LIBKML_BOTTOMTFOV_FIELD
LIBKML_NEAR_FIELD
LIBKML_SHAPE_FIELD
VSIS3_CHUNK_SIZE
VSIOSS_CHUNK_SIZE

Reply:
> GDAL_VRT_TRUSTED_MODULES

should be GDAL_VRT_PYTHON_TRUSTED_MODULES. cf ../frmts/vrt/vrtderivedrasterband.cpp

> GDAL_PDF_DPI
> GDAL_PDF_NEATLINE
> GDAL_PDF_BANDS
> GDAL_PDF_LAYERS
> GDAL_PDF_LAYERS_OFF

dynamically requested from open options. cf "grep -r alt_config_option ../frmts/pdf"

> GDAL_USER_PWD

there must be some mismatch. It should be PDF_USER_PWD if I trust "alt_config_option='PDF_USER_PWD'/>"

> VSIS3_CHUNK_SIZE
> VSIOSS_CHUNK_SIZE
dynamically generated. cf ../port/cpl_vsil_s3.cpp:            (std::string("VSI") + poFS->GetDebugKey() + "_CHUNK_SIZE").c_str(),

> CARTO_PAGE_SIZE

cf ogr_carto.h

> LIBKML_USE_SCHEMADATA

no idea about that one

> LIBKML_BOTTOMTFOV_FIELD

typo, should be LIBKML_BOTTOMFOV_FIELD

> LIBKML_NEAR_FIELD

typo, should be LIBKML_NEARFOV_FIELD

> LIBKML_SHAPE_FIELD

perhaps LIBKML_PHOTOOVERLAY_SHAPE_FIELD

Reply:
LIBKML_USE_SCHEMADATA

no idea about that one

Looks like it should be `LIBKML_USE_SIMPLEFIELD` (c8e9b2ed519dd9d9ebcbf7cae0db1d56217c9eec)